src/components/follow-in-fov.js

```diff
@@ -25,6 +25,26 @@ AFRAME.registerComponent("follow-in-fov", {
     const obj = this.el.object3D;
     const target = this.data.target.object3D;
 
+    let isHovered = false;
+    const interaction = AFRAME.scenes[0].systems.interaction;
+    const hoveredEl = interaction.state.rightRemote.hovered;
```

Comment:
i took this from `hover-menu-system`, is that not right?

Reply:
It's correct for the right remote, but now that we have a left haned remote too you should check both. Here `hover-menu-system` also checks the leftRemote:
https://github.com/mozilla/hubs/blob/d4bca2f754b85f5011db6e03b51ec733142295ef/src/systems/hover-menu-system.js#L45

Note that `hover-menu-system` has a minor bug right now because it treats both remotes entirely separately but the hover state should probably be combined. That's what the TODO in there is about. It's not a big deal because only one remote is visible at a time, but if for example you hovered on a menu with your right remote, then your left remote, then moved your right remote off it again, then the menu would not act like it was hovered even though it should. I meant to go back and fix this, even if no one notices the problem (because only one remote is active at any given time).

Reply:
ah whoops, should have read further 🤦‍♂